builder/amazon/common/run_config.go

```diff
@@ -113,6 +114,11 @@ func (c *RunConfig) Prepare(ctx *interpolate.Context) []error {
 		errs = append(errs, fmt.Errorf("An instance_type must be specified"))
 	}
 
+	if c.BlockDurationMinutes != nil && c.BlockDurationMinutes % 60 != 0 {
```

Comment:
🌵  Ah, careful c.BlockDurationMinutes is an int64 and cannot be nil.

You can run `make test` to verify.

Reply:
Good point.

Reply:
As a note, it is not handy to validate changes as current master is failing several tests.